go/pkg/bertyprotocol/store_metadata_index.go

```diff
@@ -289,7 +289,12 @@ func (m *metadataStoreIndex) areSecretsAlreadySent(pk crypto.PubKey) (bool, erro
 		return false, errcode.ErrInvalidInput.Wrap(err)
 	}
 
-	_, ok := m.sentSecrets[string(key)]
+	ownPK, err := m.ownMemberDevice.device.Raw()
+	if err != nil {
+		return false, errcode.ErrSerialization.Wrap(err)
+	}
+
+	_, ok := m.sentSecrets[string(ownPK)+string(key)]
```

Comment:
Same thing here: why add our device key to the key of a map that only concerns our current device?